spec/models/promotion/integration_spec.rb

```diff
@@ -51,6 +51,44 @@
     end
   end
 
+  context "with two promotions that should stack", :pending do
+    let(:rule) { SolidusFriendlyPromotions::Rules::LineItemProduct.new(products: [shirt]) }
+
+    let(:shirt) { create(:product, name: "Shirt", price: 30) }
+    let(:pants) { create(:product, name: "Pants", price: 40) }
+
+    let!(:distributed_amount_promo) do
+      create(:friendly_promotion,
+        :with_adjustable_action,
+        preferred_amount: 10.0,
+        apply_automatically: true,
+        lane: :post,
+        calculator_class: SolidusFriendlyPromotions::Calculators::DistributedAmount)
+    end
+    let(:promotion) { create(:friendly_promotion, name: "20% off everything", apply_automatically: true) }
```

Comment:
The name should be "20% off shirts" right?

Reply:
Fixed